### PR TITLE
Add `match` command instead `-m, --match` options

### PR DESCRIPTION
**Pull request by arborik**

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -30,6 +29,21 @@ try {
...
+    .description('prints modules which matches the provided string')
+    .action(() => {
+      const workspace = Workspace.loadSync();
+      const arg = program.args[0];
+      const { match } = arg;
+
+      if (match) {
+        console.log(matchAction(workspace, arg));
+      } else {
+        fuzzySearchAction(workspace);
```
Comment (ranyitz) [inline src/cli.js]: You can use [commander syntax](https://github.com/tj/commander.js/#command-specific-options) to validate and get the `match` string.

For example:
```js
	.command('match <string>')
    .description('prints modules which matches the provided string')
    .action(string => {
      const workspace = Workspace.loadSync();
      console.log(matchAction(workspace, string));
    });
```

Why the `fuzzySearchAction` is there?

Comment (arborik) [reply src/cli.js]: Good comment.

`fuzzySearchAction` need when string `undefined`.
If `--match` options hasn't arguments, run `fuzzySearchAction`.
Match options worked same.
Do delete `fuzzySearchAction`?

Comment (arborik) [reply src/cli.js]: Yes, you right.
Done.

File: src/cli.js [hunk 2]
```diff
@@ -30,6 +29,17 @@ try {
       console.log(listAction(workspace));
     });
 
+  program
+    .command('match <string>')
+    .description('prints modules which matches the provided string')
+    .action(string => {
+      const workspace = Workspace.loadSync();
+
+      if (string) {
```
Comment (ranyitz) [inline src/cli.js]: I don't think that `string` can be falsy, due to the `<>` of commander, maybe we should remove this if?

**Pull request merged**